### DISCUSSION TITLE
Routing Rules

### DISCUSSION THREAD

**Discussion by miroslavskela** [postalserver/postal#1798] [Help with using Postal] [answered]
Hello,
is there any possibility to create some rules for routing mails to some address endpoints. For Example, I want to route mails that comes to mail1@domain1.com  address, to mail1@gmail.com, but if sender is mail1@domain2, I don't want to route that messages to @gmail.

Thank you.

**Comment by willpower232** [comment, marked as answer]
You should be able to add the domain domain1.com and create an address endpoint for mail1@gmail.com then you can create as many routes as you want to pass the email through.

**Reply by miroslavskela** [reply]
Thanks, that is ok, I did exactly that, and it works perfectly. Is it maybe possible to filter messages that come to some address@domain1.com, and to decide should message be forwarded to @gmail account?

**Reply by willpower232** [reply]
That is not a feature of Postal I am afraid.
